resources/language/English (US)/strings.po

```diff
@@ -643,3 +643,11 @@ msgstr "Trakt - Added to watchlist"
 msgctxt "#32166"
 msgid "Trakt - Failed adding to watchlist"
 msgstr "Trakt - Failed adding to watchlist"
+
```

Comment:
Only this particular language po, or all language strings? What would you recommend?